source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+
+----
+
+##########################################
+Choosing the Right Mobile Deployment Model
+##########################################
+
+At this point you should have read through the deployment primer. It provides a large amount of context for the principles and best practices that follow. This next section aims to help you choose one of two recommended deployment models.
+
+*****************************************
+Using the Public App Stores (Recommended)
+*****************************************
+
+**Desired Outcomes:**
+
+- Quick rollout of the Mattermost mobile application.
+- Allow users to install the application on their device.
+- Ensure a high level of security and controlled access.
+- Use existing, internal processes and tools.
+
+We recommend either of these options if you are…
+
+- Testing out the mobile applications
+- Deploying Team Edition servers using no push notifications, or push notifications from Mattermost’s `TPNS <https://docs.mattermost.com/overview/faq.html#tpns>`_ (Test Push Notification Service)
+- Deploying Enterprise Edition servers using push notifications from Mattermost’s `HPNS <https://docs.mattermost.com/mobile/mobile-hpns.html>`_ (Hosted Push Notification Service)
+
+The mobile applications provided by Mattermost work with our hosted version of the Mattermost Push Notification Service (MPNS). This represents the easier path. The Mattermost mobile applications can be deployed with or without an EMM provider. These options are explained in more detail below. 
+
+    **Note:** HPNS is compatible with EMM providers.
+
+**Option 1** - Public App Store Installation (Easiest)
+
+- Users download application via public app stores (e.g. iOS App Store or Google Play)
+- Users enter URL to your hosted Mattermost Server.
+
+**Advantages:** Very easy, mobile deployment can be done by each individual user 
+**Disadvantages:** Less control over user security, no app customization
+
+**Option 2** - Public App Store Installation with EMM Provider (Easy)
+
+- Users download application via public app stores (e.g. iOS App Store or Google Play)
+- Users enter URL to your hosted Mattermost Server.
+- Extend your organization’s security best practices/requirements via your EMM provider.
+
+**Advantages:** Easy, mobile deployment can be done by each individual user, app security and configuration can be maintained via your EMM provider 
+**Disadvantages:** No app customization
+
+***********************************************
+Distributing Custom Versions of the Mobile Apps
+***********************************************
+
+**Desired Outcomes:**
+
+- Maintain full control over the distribution of applications
+- Change the look, feel, and capabilities of the Mattermost mobile application.
+- Limit message traffic from passing through the HPNS
```

Comment:
I would say something along the lines of `Use your own MPNS`, limit message traffic sounds that they can somewhat control what gets sent through HPNS which is totally false.

Reply:
Agree. The current phrase is incorrect open for misinterpretation.

Reply:
+1